Soccar [Main-Server]/src/nl/soccar/mainserver/ui/Main.java

```diff
@@ -15,7 +15,7 @@
 public class Main implements Runnable {
 
     private static final Logger LOGGER = Logger.getLogger(Main.class.getSimpleName());
-    
+
     private static final String CONNECT_GAMESERVERS_COMMAND = "connect";

```

Comment:
Deze constante string wordt niet gebruikt, moet deze verwijderd worden? Of ga je deze in een latere versie gebruiken.